Apps/AlbyHub/docker-compose.yml

```
@@ -10,7 +10,7 @@ services:
       LOG_EVENTS: true
     command: []
     container_name: albyhub
-    image: ghcr.io/getalby/hub:v1.10.2
+    image: ghcr.io/getalby/hub:v1.11.3
```

```suggestion
    image: ghcr.io/getalby/hub:v1.12.0
```

v1.12.0 was just released which contains some fixes.